spaces/S000174/properties/P000112.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000174

```

Comment:
To add proofs for this space, we should add the details of its construction to the space's description, at least enough details to verify anything we prove beyond "see this citation".

Reply:
Similar for S179

Reply:
Thanks for the suggestion. Both constructions are somewhat long, so I might need to skip some details. I'll let you review once I'm done with that.